packages/api/src/jobs/bulk_action.ts

```diff
@@ -27,6 +27,7 @@ export const bulkAction = async (data: BulkActionData) => {
   for (let offset = 0; offset < count; offset += batchSize) {
     const searchArgs = {
       size: batchSize,
+      includePending: true,
```

Comment:
oh thanks, i was going to ask for this

Reply:
Yeah, I found this bug while testing the content-fetch service